sci-mathematics/msieve/msieve-1.53.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Library for Factoring Large Integers"
+DESCRIPTION="C library implementing a suite of algorithms to factor large integers"
+HOMEPAGE="https://sourceforge.net/projects/msieve/"
+COPYRIGHT="Public Domain"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/msieve/msieve/Msieve%20v$portVersion/msieve${portVersion/./}_src.tar.gz"
+CHECKSUM_SHA256="c5fcbaaff266a43aa8bca55239d5b087d3e3f138d1a95d75b776c04ce4d93bb4"
+PATCHES="msieve-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	msieve$secondaryArchSuffix = $portVersion
+	cmd:msieve$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libecm$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	msieve${secondaryArchSuffix}_devel = $portVersion
+	devel:libmsieve$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	msieve$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libecm$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+PATCH()
+{
+	# Allow owns CFLAGS, could be useful when doing debug builds
+	sed -e 's/^CFLAGS =/CFLAGS +=/' -i Makefile
+	# Remove -march=native, while this flag provides speedup,
+	# it's not suitable for binary distribution
+	sed -e 's/-march=native//' -i Makefile
```

Comment:
Good catch, about the `-march=native`.

Hmm, since the 2 sed commands change the same file, it might be a good idea to change PATCH to:
```sh
PATCH()
{
	# - Allow our own CFLAGS; could be useful when doing debug builds.
	# - Remove -march=native; while this flag provides speedup,
	#   it's not suitable for binary distribution.
	sed -i \
		-e 's/^CFLAGS =/CFLAGS +=/' \
		-e 's/-march=native//' \
		Makefile
}
```
I don't know if this would be an improvement, but this variant looks a little bit more eye-friendly.
What do you think ?
Everything else is OK, so we could even merge as is if you prefer.

Reply:
I personally like having them separated for no reason other than I hate tabs :P. So let's just merge them as is

Reply:
OK, that's fine too.